owncloudApp/src/main/java/com/owncloud/android/presentation/files/removefile/RemoveFilesDialogFragment.kt

```diff
@@ -106,7 +104,7 @@ class RemoveFilesDialogFragment : ConfirmationDialogFragment(), ConfirmationDial
                     R.string.confirmation_remove_files_alert
                 }
             }
-            val localRemoveButton = if (!containsAvailableOffline && isAvailableLocally) {
+            val localRemoveButton = if (isAvailableLocallyAndNotAvailableOffline) {
```

Comment:
The situation is that what is checked in that if has already been checked previously in this Use Case `IsAnyFileAvailableLocallyAndNotAvailableOfflineUseCase`, so it would no longer be necessary to keep that check there. I am talking about this one:
`val localRemoveButton = if (!containsAvailableOffline && isAvailableLocally) {`
Therefore, since we are already bringing the variable that has validated these two variables previously, it would be redundant to keep this condition as it was. We are saving code and not repeating a check that was already done previously.
This check is also not necessary because it was already done previously:
```
if (file.isAvailableOffline) {
                    containsAvailableOffline = true
                }
```